pandas/tests/frame/test_block_internals.py

```diff
@@ -40,17 +40,17 @@ def test_cast_internals(self):
 
     def test_consolidate(self):
```

Comment:
can you add a single test (test_consolidate_deprecation), which uses ``assert_produces_warning(FutureWarning)``